Slack.Automation/Promact.Erp.Core/ActionFilters/ApiExceptionLoggerFilter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Autofac.Extras.NLog;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http.Filters;
+
+namespace Promact.Erp.Core.ActionFilters
+{
+    public class ApiExceptionLoggerFilter : IExceptionFilter
+    {
+        private readonly ILogger _logger;
+
+        public ApiExceptionLoggerFilter(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public bool AllowMultiple { get { return true; } }
+
+        public Task ExecuteExceptionFilterAsync(
+                HttpActionExecutedContext actionExecutedContext,
+                CancellationToken cancellationToken)
+        {
+            _logger.Error("Web Service Error Message:" + actionExecutedContext.Exception.Message);
+            _logger.Error("Web Service Error StackTrace:" + actionExecutedContext.Exception.StackTrace);
+            return Task.FromResult(0);
```

Comment:
IExceptionFilter has method "ExecuteExceptionFilterAsync" and its return type is task so I return task.